### PR TITLE
Add `setup` command + `setup docker` subcommand to nest CLI

### PR DESCRIPTION
**Pull request by hello-smile6**
This PR adds a `setup` command with a `docker` subcommand that configures rootless Docker to the nest CLI.

### CODE REVIEW

File: cli/cli.rb [hunk 1]
```diff
@@ -49,12 +49,30 @@ def self.exit_on_failure?
...
+    end
+  end
+
   class Nest < Thor
     desc "subdomain SUBCOMMAND ...ARGS", "manage your nest subdomains"
     subcommand "subdomain", Subdomain
 
     desc "domain SUBCOMMAND ...ARGS", "manage your nest custom domains"
     subcommand "domain", Domain
+    desc "setup SUBCOMMAND ...ARGS", "set something up on your nest account"
```
Comment (polypixeldev) [inline cli/cli.rb]:
```suggestion
    desc "setup SUBCOMMAND ...ARGS", "setup a tool to use on nest"
    subcommand "setup", Setup
```
I think this wording is better, and missing the subcommand

**Review by polypixeldev — CHANGES_REQUESTED** [review]
just a couple things!

**Review by polypixeldev — APPROVED** [review]
LGTM!

**Comment by polypixeldev** [comment]
if there's nothing else for this PR, can you un-draft it?

**Comment by hello-smile6** [comment]
This should be ready to merge if someone would be willing to

**Pull request merged**